taxonomies/categories.txt

```diff
@@ -687,7 +687,7 @@ ciqual_food_name:fr:Plat légumes, avec féculent et viande/poisson, dès 6-8 mo
 <en:Main meals for babies
 <en:From 6 months
 en:Vegetable dish for baby with starch and meat
-fr:Plat avec légumes et féculents et viande
+fr:Plat aux légumes pour bébés, avec féculents et viande
```

Comment:
```suggestion
fr:Plat aux légumes pour bébés avec féculents et viande
```
commas will be handed as synonyms

Reply:
also, since it's 3 synonyms, we could have the number of months ?

Reply:
I've tried updating my pull request, and squashing the commit, but I'm not sure I did it right.